types/node/index.d.ts

```diff
@@ -4975,9 +4969,15 @@ declare module "tls" {
         passphrase?: string;
         cert?: string | Buffer;
         ca?: string | Buffer;
-        crl?: string | string[];
         ciphers?: string;
         honorCipherOrder?: boolean;
+        ecdhCurve?: string;
+        clientCertEngine?: string;
+        crl?: string | string[];
```

Comment:
According to doc this should be `crl?: string | Buffer | Array<string | Buffer>`

Reply:
fixed